src/bucketd/processor.rs

```diff
@@ -566,13 +568,9 @@ impl Runtime {
 
     pub(super) fn finalize_transfer(
```

Comment:
After @cryptohunter's comment above, I think it better let's remove one of the methods, as there is no need to keep duplicate methods. 

Also, I would like to rename `finalize_transfer` to `finalize_transfers` to use `NewTransfer` instead of `Vec<(StateTransfer, Vec<SealEndpoint>)>`, because it is more compatible with cli and directly method call.

Does make sense to you?

Reply:
I'm afraid it doesn't. There is no API method in rgb-node that handles CLI stuff and I think that this should be respected.

In the `NewTransfer` object you are adding a `consignment_out` that is not used nor needed in rgb-node in order to finalize the consignment. The ability to save the finalized consignment to a path is a CLI feature and therefore it should be implemented there. This is because writing the finalized consignment to file should be optional and handled by the libraries that communicate with rgb-node.

About CLI compatibility, I can't see any issue using the already existing `finalize_transfers` method since `FinalizeTransfersRes` will contain a vector of consignments that will be in the same order as the `endseals` vector in `TransferCommand::Finalize`.

Reply:
> In the NewTransfer object you are adding a consignment_out that is not used nor needed in rgb-node in order to finalize the consignment. The ability to save the finalized consignment to a path is a CLI feature and therefore it should be implemented there. This is because writing the finalized consignment to file should be optional and handled by the libraries that communicate with rgb-node.

Hi @zoedberg, thanks for feedback, I agree with you.